packages/runtimes/hmr/src/loaders/hmr-runtime.js

```diff
@@ -241,10 +241,13 @@ function updateLink(link) {
       link.parentNode.removeChild(link);
     }
   };
+  var newLinkUrl = link.getAttribute('href').replace(/[?&]__parcel__timestamp__=\d+/, '');
+  newLinkUrl += (newLinkUrl.indexOf('?') < 0 ? '?' : '&') + '__parcel__timestamp__=' + Date.now()
```

Comment:
Maybe instead of reloading all `<link>` elements across all domains, we could only reload the css from the current domain (e.g. parcel dev server). Might avoid unnecessary network requests. Would that also solve your issue?